### PR TITLE
Bug 1797593 - Pre-land text for cookie banner handling dialog

### PR DESCRIPTION
**Pull request by Amejia481**
### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### QA

- [x] **QA Needed**

### To download an APK when reviewing a PR (after all CI tasks finished running):
1. Click on `Checks` at the top of the PR page.
2. Click on the `firefoxci-taskcluster` group on the left to expand all tasks.
3. Click on the `build-debug` task.
4. Click on `View task in Taskcluster` in the new `DETAILS` section.
5. The APK links should be on the right side of the screen, named for each CPU architecture.

### GitHub Automation

Used by GitHub Actions.

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -341,6 +341,26 @@
     <string name="reduce_cookie_banner_details_panel_description_off_for_site">%1$s will clear this site’s cookies and refresh the page. Clearing all cookies may sign you out or empty shopping carts.</string>
     <!-- Long text for a detail explanation indicating what will happen if cookie banner handling is on for a site, this is shown as part of the cookie banner panel in the toolbar. The first and second parameter are the application name -->
     <string name="reduce_cookie_banner_details_panel_description_on_for_site">%1$s can try to automatically reject cookie requests. If a reject option isn’t available, %2$s may accept all cookies to dismiss the banner.</string>
+    <!-- Title text for the onboarding dialog use on the control branch of the experiment to determine which context users engaged the most-->
```
Comment (iorgamgabriel) [inline app/src/main/res/values/strings.xml:344]: cookie banner dialog not onboarding dialog . I think

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -341,6 +341,26 @@
...
+    <string name="reduce_cookie_banner_control_experiment_dialog_title">Cookie banners begone!</string>
+    <!-- Body text for the onboarding dialog use on the control branch of the experiment to determine which context users engaged the most-->
```
Comment (gabrielluong) [inline app/src/main/res/values/strings.xml:346]:
```suggestion
    <!-- Body text for the onboarding dialog use on the control branch of the experiment to determine which context users engaged the most -->
```
Could you add a space before `-->` for all of these?

File: app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Amejia481) [reply app/src/main/res/values/strings.xml]: Updated :) !

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (Amejia481) [reply app/src/main/res/values/strings.xml]: Updated :) !

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -341,6 +341,26 @@
...
     <!-- Long text for a detail explanation indicating what will happen if cookie banner handling is on for a site, this is shown as part of the cookie banner panel in the toolbar. The first and second parameter are the application name -->
     <string name="reduce_cookie_banner_details_panel_description_on_for_site">%1$s can try to automatically reject cookie requests. If a reject option isn’t available, %2$s may accept all cookies to dismiss the banner.</string>
+    <!-- Title text for the dialog use on the control branch of the experiment to determine which context users engaged the most -->
+    <string name="reduce_cookie_banner_control_experiment_dialog_title" tools:ignore="UnusedResources">Cookie banners begone!</string>
+    <!-- Body text for the dialog use on the control branch of the experiment to determine which context users engaged the most -->
+    <string name="reduce_cookie_banner_control_experiment_dialog_body" tools:ignore="UnusedResources">Automatically reject cookie requests, when possible. Otherwise, accept all cookies to dismiss cookie banners.</string>
+    <!-- Remind me later text button for the onboarding dialog-->
```
Comment (gabrielluong) [inline app/src/main/res/values/strings.xml:348]:
```suggestion
    <!-- Remind me later text button for the onboarding dialog -->
```

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -341,6 +341,26 @@
...
+    <!-- Change setting text button, for the dialog use on the control branch of the experiment to determine which context users engaged the most -->
+    <string name="reduce_cookie_banner_control_experiment_dialog_change_setting_button" tools:ignore="UnusedResources">Dismiss banners</string>
+    <!-- Title text for the dialog use on the variant 1 branch of the experiment to determine which context users engaged the most -->
+    <string name="reduce_cookie_banner_variant_1_experiment_dialog_title" tools:ignore="UnusedResources">See fewer cookie pop-ups</string>
+    <!-- Body text for the dialog use on the variant 1 branch of the experiment to determine which context users engaged the most -->
+    <string name="reduce_cookie_banner_variant_1_experiment_dialog_body" tools:ignore="UnusedResources">Automatically answer cookie pop-ups for distraction-free browsing. Firefox will reject all requests if possible, or accept all if not.</string>
+    <!-- Change setting text button, for the onboarding dialog use on the variant 1 branch of the experiment to determine which context users engaged the most-->
```
Comment (gabrielluong) [inline app/src/main/res/values/strings.xml:356]:
```suggestion
    <!-- Change setting text button, for the onboarding dialog use on the variant 1 branch of the experiment to determine which context users engaged the most -->
```

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (Amejia481) [reply app/src/main/res/values/strings.xml:348]: Addressed!

File: app/src/main/res/values/strings.xml (hunk 4 above)
Comment (Amejia481) [reply app/src/main/res/values/strings.xml:356]: Addressed!

**Review by iorgamgabriel — APPROVED** [review]

**Pull request merged**